Description: Make rc1 library names look like the release version.
Author: A. Maitland Bottoms <bottoms@debian.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,7 +50,7 @@
 # Set the version information here
 set(VERSION_INFO_MAJOR_VERSION 3)
 set(VERSION_INFO_API_COMPAT    7)
-set(VERSION_INFO_MINOR_VERSION 9rc1)
+set(VERSION_INFO_MINOR_VERSION 9)
 set(VERSION_INFO_MAINT_VERSION 0)
 include(GrVersion) #setup version info
 
